recipe-server/client/control_new/utils/api.js

```diff
@@ -0,0 +1,75 @@
+import * as Cookie from 'js-cookie';
+
+
+export default class APIClient {
```

Comment:
Sure. To me there was nothing inherently wrong with the way we were doing it. I do however think this is: a) cleaner to read. b) more flexible should we for some reason need to call an API with a different root. c) in the future we could create shortcut methods like `.post`, `.get`, `.patch` etc. and perhaps then consider making some changes to the requests action creator. 

There's not a lot of immediate benefit to this change but to my mind there are only potential upsides with no downsides.